info/v1/container.go

```diff
@@ -296,6 +296,14 @@ type CpuUsage struct {
 	// Time spent in kernel space.
 	// Unit: nanoseconds.
 	System uint64 `json:"system"`
+
+	// CPU time consumed per core in kernel mode
+	// Units: nanoseconds.
+	PercpuUsageInKernelmode []uint64 `json:"percpu_usage_in_kernelmode"`
+
+	// CPU time consumed per core in user mode
+	// Units: nanoseconds.
+	PercpuUsageInUsermode []uint64 `json:"percpu_usage_in_usermode"`
```

Comment:
The same as above, consider changing to `PerCpuUser` and "json:"percpu_usage_user".